daemon/meshnet/handler.go

```diff
@@ -250,3 +253,118 @@ func (m *Meshnet) Update(ctx context.Context, pod *mpb.RemotePod) (*mpb.BoolResp
 	}
 	return &mpb.BoolResponse{Response: true}, nil
 }
+
+//------------------------------------------------------------------------------------------------------
+func (m *Meshnet) RemGRPCWire(ctx context.Context, wireDef *mpb.WireDef) (*mpb.BoolResponse, error) {
+
+	if err := grpcwire.DeleteWiresByPod(wireDef.KubeNs, wireDef.LocalPodName); err != nil {
+
+		return &mpb.BoolResponse{Response: false}, err
+	}
+	return &mpb.BoolResponse{Response: true}, nil
+}
+
+//------------------------------------------------------------------------------------------------------
+func (m *Meshnet) AddGRPCWireLocal(ctx context.Context, wireDef *mpb.WireDef) (*mpb.BoolResponse, error) {
+
+	locInf, err := net.InterfaceByName(wireDef.VethNameLocalHost)
+	if err != nil {
+		log.Errorf("Failed to retrieve interface ID for interface %v. error:%v", wireDef.VethNameLocalHost, err)
+		return &mpb.BoolResponse{Response: false}, err
+	}
+
+	//Using google gopacket for packet receive. An alternative could be using socket. Not sure it it provides any advantage over gopacket.
```

Comment:
If I understand this correctly, if you use a socket, you become the ultimate destination for those packets.
In case of a pcap, they would have to be dropped by the host OS due to no listening socket. 
How hard would it be to refactor this to use a standard listening socket?

Reply:
In this case, once the grpc wire is create then only pacp kicks in for that specific interface. So for the lifetime of a wire, meshnet daemon is always listening and passing the received packet to the other end of the wire. So the OS does not have to drop anything as some is always listening as log as the grpc wire is alive. 

A raw socket based implementation is possible. Its some work but not a lot.  gopacket/pcap is around 5~10 lines and raw socket will bigger that that but not a lot of code.  Checked with pcap source code, internally it uses socket too. 

Let me discus this little more with you on slack and we will put the summary here.

Reply:
yep, makes sense. I'm all for using pcap, especially if it makes it easier, but I want to understand the implications of doing so. Based on my understanding, `google/gopacket/pcap` relies on `libpcap`, which receives a copy of all packets, but the actual packets remain intact and continue through the networking stack, in which case they would get dropped inside the meshnetd.  am I misunderstanding something?

Reply:
just noticed your note about slack. let's discuss there first.